doc/source/user_guide/skimage2_migration.md

```diff
@@ -8,24 +8,85 @@ This document is a work in progress and still subject to change.
+Run the following [warnings filter](https://docs.python.org/3/library/warnings.html#the-warnings-filter) before you use scikit-image in your code:
+
+```python
+import warnings
+import skimage as ski
+warnings.filterwarnings(action="default", category=ski.util.PendingSkimage2Change)
+```
+
+This will raise a warning for any piece of code which needs updating to continue working with the new (skimage2) API the exact same way it used to (in the old API).
+
+## Updating existing code
+
+When switching to the new namespace in version 2.0, some code will need updating to keep working as before.
+
+:::{note}
+Because these changes live in a new namespace, your code will _not_ break automatically if you don't explicitly change your imports and start importing from `skimage2`!
+:::
+
+(threshold-blob-funcs)=
+
+### Range-preserving thresholding in `feature.blob_*` functions
+
+**Affects:**
+
+- {func}`skimage.feature.blob_dog`
+- {func}`skimage.feature.blob_doh`
+- {func}`skimage.feature.blob_log`
+
+**Description:**
+
+Starting with **version 0.26**, the value range of the input `image` is always preserved.
+When `image` had an integer dtype, its value range was scaled -- unsigned integers to the interval [0, 1], signed integers to [-1, 1].
+This affected the now deprecated `threshold` parameter, whose absolute value would compare differently with a scaled or preserved value range.
+In other words, the same `threshold` value would have different effects depending on `image.dtype`.
+With this deprecation, the new parameter `threshold_abs` is introduced.
+It will always be compared against a range-preserving version of `image` and behave consistently regardless of `image.dtype`.
+
+Starting with **version 2.0** and the new `skimage2` namespace, the default values of these parameters are updated too.
+The default of `threshold` will be set to `None` and `threshold_rel` will be set to the old value of `threshold`.
+Calls to these functions that rely on default values may change behavior, so we recommend setting `threshold_abs` and `threshold_rel` explicitly.
+
+In **version 2.2** (or later), the old `threshold` parameter will be removed completely.
+
+**How to update:**
+
+To discover all calls that need to be updated, [enable skimage2-related warnings](#enable-skimage2-warnings).
+
+Since the old behavior dtype-specific behavior might have been unexpected, we
+suggest setting `threshold_abs=None` and finding an appropriate relative value
+for `threshold_rel`.
+
+However, if you want to maintain the exact old behavior across all releases:
+
+- Set `threshold_rel=None` (to ignore the new default in version 2.0).
+- Set `threshold_abs` to the old value of `threshold`.
+- Wrap the input to the `image` parameter with {func}`skimage.util.img_as_float`
+  before passing it to the function. This will conditionally rescale the input
+  range of the `image` as before depending on its dtype.
 
-## Changes pre skimage2
+## Changes unrelated to skimage2
```

Comment:
Not sure what this heading is meant to communicate?

Reply:
It's meant to list the planned API changes, doable independent of skimage2 (cf. https://github.com/scikit-image/scikit-image/wiki/API-changes-for-skimage2#proposed-api-changes-doable-independent-of-skimage2).

Reply:
I think I intended to do so when I started working on this. With the current approach I don't think it's strictly necessary. But I think it would help being more transparent about what kind of normal deprecations are coinciding with the skimage2 transition.

I'm pretty sure we are missing some that we will introduce with the next release and that will be completed after 2.0 (e.g. the `from_estimate` stuff). But I'd prefer to keep this outside this PRs's scope.